validation-test/Sema/type_checker_perf/fast/rdar46713933_concrete_args.swift

```diff
@@ -7,8 +7,5 @@ func wrap<T: ExpressibleByFloatLiteral>(_ key: String, _ value: T) -> T { return
 func wrap<T: ExpressibleByStringLiteral>(_ key: String, _ value: T) -> T { return value }
 
 func wrapped(i: Int) -> Int {
-  // FIXME: When this stops being "too complex", turn the integer value into
-  // an integer literal.
-  // expected-error@+1{{reasonable time}}
```

Comment:
Heh, should we try doing what the FIXME suggests as well?

Reply:
Yeah, I have it as - https://github.com/apple/swift/blob/main/validation-test/Sema/type_checker_perf/slow/rdar46713933_literal_arg.swift. Still slow unfortunately since it can't use `Int` early (yet).